src/Lock.php

```diff
@@ -492,20 +581,33 @@ public static function showForItem(CommonDBTM $item)
         foreach ($DB->request(['glpi_networknames', 'glpi_networkports'], $params) as $line) {
             $networkname->getFromDB($line['id']);
             if ($first) {
-                echo "<tr><th colspan='2'>" . NetworkName::getTypeName(Session::getPluralNumber()) . "</th></tr>\n";
+                echo "<tr>";
+                echo "<th width='10'></th>";
+                echo "<th>" . $networkname->getTypeName(Session::getPluralNumber()) . "</th>";
+                echo "<th>" . __('FQDN') . "</th>";
+                echo "<th></th>";
+                echo "<th>" . __('Automatic inventory') . "</th>";
+                echo "</tr>";
                 $first = false;
             }
 
             echo "<tr class='tab_bg_1'>";
-
             echo "<td class='center' width='10'>";
             if ($networkname->can($line['id'], UPDATE) || $networkname->can($line['id'], PURGE)) {
                 $header = true;
                 echo "<input type='checkbox' name='NetworkName[" . $line['id'] . "]'>";
             }
             echo "</td>";
+            echo "<td class='left'>" . $networkname->getLink() . "</td>";
 
-            echo "<td class='left' width='95%'>" . $networkname->getName() . "</td>";
+            $fqdn = new FQDN();
+            $fqdn_name = "";
+            if ($fqdn->getFromDB($networkname->fields['fqdns_id'])) {
+                $fqdn_name = $fqdn->fields['name'];
+            }
+            echo "<td class='left'></td>";
+            echo "<td class='left'>" . $fqdn_name . "</td>";
```

Comment:
Header is contained in the 3rd `th`.
```suggestion
            echo "<td class='left'>" . $fqdn_name . "</td>";
            echo "<td class='left'></td>";
```

Reply:
fixed